javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Currency calculator</title>
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1>Currency calculator</h1>
+        </header>
+
+        <form action="#">
+            <div class="margin">
+                <label for="amount">Amount to convert: </label> <br>
+                <input type="number" id="amount" name="amount" value="1">
```

Comment:
you might want to add `min="0"` to this input, now it allows negative numbers. Nice idea with initial value set to 1 with the currency converter